app/react/kubernetes/cluster/ConfigureView/ConfigureForm/validation.ts

```diff
@@ -23,14 +23,15 @@ const storageClassFormValuesSchema = array()
     })
   )
   .test(
-    // invalid if any storage class is not selected or if it's selected and at least one access mode is selected
+    // invalid if any storage class is not selected or
+    // if it's selected and at least one access mode is selected
     'accessModes',
     'Shared access policy configuration required.',
     (storageClasses) => {
       const isValid = storageClasses?.every(
         (value) =>
           !value.selected ||
-          value.AccessModes?.some((accessMode) => accessMode.selected)
+          (value.AccessModes && value.AccessModes?.length > 0)
```

Comment:
can remove `value.AccessModes && `

Reply:
no length on undefined is giving error so I added it.